components/raftstore/src/store/peer.rs

```diff
@@ -1076,6 +1079,7 @@ where
             skip_bcast_commit: true,
             pre_vote: cfg.prevote,
             max_committed_size_per_ready: MAX_COMMITTED_SIZE_PER_READY,
+            priority: if peer.is_witness { 0 } else { 1 },
```

Comment:
I'm afraid this make election fail. For example, there are 5 replicas, and in the loop of rolling update. 3 of them are using the new binary, so their priority is 1. the remaining are 0. If PD transfer leader to one of the remaining node, the operation will fail and cause unavailability.

Reply:
Seems the only way to solve that is changing the raft-rs definition, lower number should mean higher priority

Reply:
PTAL https://github.com/tikv/raft-rs/pull/493

Reply:
remove setting priority in this PR, will support in next PRs

Reply:
PTAL